view/frontend/web/js/model/step-navigator-mixin.js

```diff
@@ -0,0 +1,22 @@
+define([
+    'mage/utils/wrapper',
+    'jquery'
+], function (wrapper, $) {
+    'use strict';
+
+    let mixin = {
+        handleHash: function (originalFn) {
+            let hashString = window.location.hash.replace('#', '');
```

Comment:
Variable is never redeclared so use of let is misleading
In M2 style JS I'd stick to `var`

Reply:
I never know which to use and assumed `let` was the more up-to-date approach 😛

Reply:
yeah it is
you could use let and const
my only feeling is they are not used else where in M2 JS - so it's just nice to keep things consistent